### PR TITLE
German Firefox Marketing guide update

### PR DESCRIPTION
**Pull request by peiying2**
Update request per Anika: https://docs.google.com/document/d/1LlCWCsMTqx8XvYszfde5DOLMqezgi4mHTNnH4QhKUfU/edit#

### CODE REVIEW

**Comment by peiying2** [comment]
@TheoChevalier don't feel obligated, if you have time, do check on the format and if I updated them all. Anika will check on the accuracy of the content update. I would like to release the update sooner than later. Thanks!

File: docs/de/firefox_marketing.md [hunk 1]
```diff
@@ -136,6 +140,7 @@ Bei Unsicherheiten kann auch die [Transvision Suche](https://transvision.mozfr.o
 |Internet health|Zustand des Internets|
 |Internet experience|Surf-Erlebnis|
 |JavaScript-Debugger|JavaScript-Debugger|
+|Join Firefox|Join Firefox<br>Komm zu Firefox(nicht auf CTA buttons nutzen, dort wird zwischen Sign in (einloggen) und Sign up (Anmelden oder Konto erstellen) differenziert.</br>|
```
Comment (TheoChevalier) [inline docs/de/firefox_marketing.md]: `Firefox(nicht` missing space

Comment (TheoChevalier) [reply docs/de/firefox_marketing.md]: Also `Join Firefox` should be removed from the German equivalent column, according to the doc

Comment (peiying2) [reply docs/de/firefox_marketing.md]: Thank you @TheoChevalier for the review and spotting the error. She added more changes in the process. I will ask her about Join Firefox.

**Pull request merged**